enterprise/validators.py

```diff
@@ -24,7 +24,7 @@ def validate_image_extension(value):
     """
     config = get_app_config()
     ext = os.path.splitext(value.name)[1]
-    if config and not ext.lower() in getattr(config, "valid_extensions", []):
+    if config and not ext.lower() in getattr(config, "valid_image_extensions", []):
```

Comment:
Nit: Given that the `valid_image_extensions` variable used here (as well as the `image_size` variable used below) is defined unconditionally on the config, do we really need the `getattr` calls here and below? I'd just go with `config.valid_image_extensions`; we don't need yet another layer of defaults.